src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -108,4 +107,13 @@ export const Checkmark = styled.div`
   width: 18px;
   height: 18px;
   border-radius: 1px;
+  position: relative;
+  svg {
+    position: absolute;
+  }
+`
+
+export const StyledHelperText = styled(HelperText)`
+  margin-top: 0;
+  margin-left: ${sizes(10)};
```

Comment:
```suggestion
  margin-left: ${sizes(9)};
```

Reply:
Are you sure? 
9:
![image](https://user-images.githubusercontent.com/51168865/114160566-5b54f200-9927-11eb-894f-3f5eb704874b.png)
10: 
![image](https://user-images.githubusercontent.com/51168865/114160628-6c9dfe80-9927-11eb-8104-0d8a0d913e59.png)

Reply:
oh, maybe I shouldn't use`sizes` here. I'll look into this again